src/cascadia/TerminalSettingsEditor/ProfileViewModel.cpp

```diff
@@ -104,150 +105,65 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
             _unfocusedAppearanceViewModel = winrt::make<implementation::AppearanceViewModel>(profile.UnfocusedAppearance().try_as<AppearanceConfig>());
         }
 
+        _parsedPadding = StringToXamlThickness(_profile.Padding());
         _defaultAppearanceViewModel.IsDefault(true);
     }
 
     void ProfileViewModel::LeftPadding(double value) noexcept
     {
-        const hstring& padding = _GetNewPadding(PaddingDirection::Left, value);
-
-        Padding(padding);
+        if (std::abs(_parsedPadding.Left - value) >= .0001)

```

Comment:
@lhecker I was so horribly indoctrinated at a young age - this can't STILL be right, can it?